### PR TITLE
Fixed: Flaky cypress tests

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

I worked on the fixing flaky cypress tests

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js [hunk 1]
```diff
@@ -13,9 +13,12 @@
 
 /// <reference types="cypress" />
 
+import { SEARCH_INDEX } from '../constants/constants';
+
 export const descriptionBox =
   '.toastui-editor-md-container > .toastui-editor > .ProseMirror';
 export const uuid = () => Cypress._.random(0, 1e6);
+const RETRY_TIMES = 30;
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/cypress/common/common.js:21]: Can we move this to the constants file?

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js [hunk 2]
```diff
@@ -403,12 +406,23 @@ export const searchEntity = (term, suggestionOverly = true) => {
...
-  interceptURL('GET', '/api/v1/search/*', 'explorePageSearch');
+  interceptURL(
+    'GET',
+    `/api/v1/search/query?q=*&from=*&size=*&index=${SEARCH_INDEX[entity]}`,
+    'explorePageTabSearch'
+  );
+  interceptURL(
+    'GET',
+    `/api/v1/search/suggest?q=*&index=dashboard_search_index,table_search_index,topic_search_index,pipeline_search_index,mlmodel_search_index`,
+    'searchQuery'
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/cypress/common/common.js:417]: Can we use the `SEARCH_INDEX` for the index value?

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js (hunk 1 above)
Comment (vivekratnavel) [inline openmetadata-ui/src/main/resources/ui/cypress/common/common.js:21]: With the retry count of 30 and a waiting time of 20 seconds between each try, we are waiting a whole 10 minutes for any ingestion job to succeed. We should not be retrying these many times in my opinion.

**Review by Sachin-chaurasiya — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress/common/common.js:21]: Hi @vivekratnavel, We are not waiting for a total of 10 min for every spec, as ingestion status update is entirely dependent on airflow ingestion we are not sure about when it is going to be successful, so we had to retry until it's successful but we are just providing an upper limit to it! also, it's old logic I have just moved the variable to an outside function so we can utilize this if needed!

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js (hunk 2 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress/common/common.js:417]: it's fine in my opinion as we need here all the index to make URL and not any particular index or dynamic index

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress/common/common.js:21]: constants file contains the data which we use across all specs, whereas `RETRY_TIMES` is related only to a particular function which is in `common.js` file

**Comment by ShaileshParmar11** [comment]
`Bots.spec.js` is already fixed by #7988

**Review by vivekratnavel — APPROVED** [review]

**Pull request merged**